examples/with-npm/package.json

```diff
@@ -1,19 +1,21 @@
 {
+  "name": "with-npm",
   "private": true,
-  "workspaces": [
-    "apps/*",
-    "packages/*"
-  ],
   "scripts": {
-    "build": "turbo run build",
-    "dev": "turbo run dev",
-    "lint": "turbo run lint"
+    "build": "turbo build",
+    "dev": "turbo dev",
+    "lint": "turbo lint",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\""
   },
   "devDependencies": {
-    "eslint-config-custom": "*",
-    "eslint": "7.32.0",
-    "prettier": "^2.5.1",
-    "turbo": "^1.9.3"
+    "@repo/eslint-config": "*",
+    "@repo/typescript-config": "*",
+    "prettier": "^3.1.0",
+    "turbo": "latest"
   },
-  "packageManager": "npm@8.19.4"
+  "packageManager": "npm@9.6.7",
```

Comment:
Are we sure about this? We're using and recommending Node 18 for the most part everywhere, which I believe ships with npm@8. If we're bumping everything to Node 20, we should make that clear everywhere